### PR TITLE
feat: add support for node 16

### PR DESCRIPTION
**Pull request by iamsobanjaved**
- Added support for node 16
- Replaced `node-sass` with `sass`
- Replaced `extract-text-webpack-plugin` with `mini-css-extract-plugin`
- Upgraded `edx/paragon`
- Transform SVG file in jest to fix tests

Ticket: [BOM-3222](https://openedx.atlassian.net/browse/BOM-3222)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -102,6 +102,10 @@
   "jest": {
     "testEnvironment": "jest-environment-jsdom-global",
     "testURL": "http://localhost/",
+    "transform": {
+      "^.+\\.jsx?$": "babel-jest",
+      "^.+\\.svg$": "./src/svgTransform.js"
```
Comment (iamsobanjaved) [inline package.json:107]: Introduced this to fix tests, from here: https://stackoverflow.com/a/59062195

File: src/svgTransform.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  process() {
```
Comment (binodpant) [inline src/svgTransform.js:2]: maybe worth a comment here on why this change?

**Review by binodpant — APPROVED** [review]
looking fine to me with the svg fix!
